### PR TITLE
Resize the width of the logo

### LINKED ISSUE #216: Add logo to docs
sub task of #203

### PR DESCRIPTION
**Pull request by Mariatta**
Make the logo fill entire nav column.

screen shot 2016-11-09 at 9 11 43 pm

Hope this works :)
Thanks.

Closes #216

### CODE REVIEW

**Comment by Adrianzatreanu** [comment]
ack c5b6035

**Comment by Adrianzatreanu** [comment]
@rultor merge

**Comment by Adrianzatreanu** [comment]
unack c5b6035

File: _static/custom.css [hunk 1]
```diff
@@ -11,7 +11,7 @@ div.sphinxsidebar .caption-text {
 }
```
Comment (Adrianzatreanu) [inline _static/custom.css]: wait, why does it close "that issue"? either create a new one or dont close any, as that one is irrelevant and already closed :D

**Comment by Mariatta** [comment]
Sorry about that, let me open a new ticket

**Comment by Mariatta** [comment]
Opened #291

File: _static/custom.css (hunk 1 above)
Comment (Mariatta) [inline _static/custom.css]: Thanks. I adjusted the commit, it will now close #291  :)

File: _static/custom.css [hunk 2]
```diff
@@ -11,9 +11,9 @@ div.sphinxsidebar .caption-text {
...
 
 img.logo {
-    width: 100px;
+    width: 220px;
```
Comment (sils) [inline _static/custom.css]: can we have % here? Does this havve to be hardcoded px?

**Comment by Mariatta** [comment]
@sils Thanks for the feedback. I changed the width to use percentage.

**Comment by sils** [comment]
ack https://github.com/coala/documentation/pull/280/commits/fa375b6d1fdebc6238c624a3193a3447913efa1f

**Comment by sils** [comment]
@Mariatta needs a rebase, then we can merge :)

**Comment by Mariatta** [comment]
rebased :) (I think)

**Comment by Nosferatul** [comment]
yes 👍

**Comment by Nosferatul** [comment]
ack 41302df

**Comment by Nosferatul** [comment]
@rultor merge

**Comment by Nosferatul** [comment]
@Mariatta you can check if you rebased by checking your branch if it's with 1 commit(commits) ahead of master.

**Pull request merged**